lib/src/experimental/built_redux_component.dart

```diff
@@ -145,6 +165,13 @@ abstract class BuiltReduxUiComponent<
   /// Related: [connectedState]
   Substate connect(V state);
 
+  /// Whether the component should be a "pure" component.
+  ///
+  /// A "pure" component will only re-render when [connectedState] is updated or [redraw] is called directly.
```

Comment:
Or when the `store` changes, right?

Should there also be a note showing usage of this? For example:
```dart
/// To enable this functionality, override this getter in a subclass to return `true`.
///
///  When using this, do not override [redraw] or [shouldComponentUpdate].
```

Reply:
Addressed